README.md

```diff
@@ -149,7 +159,18 @@ The ancestors have provided an example of one route; you do the other six!
 GET '/warriors/:id'
   * This is the show route, which finds a warrior by ID, and displays information about that warrior.
 
-Replace this with your answer
+POST 'warriors/:id'
+* this it the route by which warriors are added
+
+put 'warriors/:id'
+*updates whole profile to new warrior
+*
+PATCH 'warriors'
+*updates just part of the warrirors profile
+
+DELETE 'warriors/:id'
+* this is the route by which warriors are deleted
+
 ```
```

Comment:
Your description of the difference between PUT and PATCH is correct, but it's worth noting that PUT and PATCH are functionally interchangeable, and both need an `:id` to tell which warrior's profile to target. You're also missing  the index, new, and edit routes.